2019-January-29/writingSamples/frida.tex

```diff
@@ -0,0 +1,141 @@
+will likely bounce back with a bunch of reviewers' comments. I wanted to
+take this opportunity to revisit my introduction and make sure it makes
+sense to the reader in terms of content and flow. I'm only including the
+abstract here for those that want more context, so don't feel like you
+have to revise it. Thank you for your thoughtful feedback!
+
+\textbf{Abstract}
+
+Sudden unexpected death in epilepsy (SUDEP) is a major cause of
+mortality in patients with drug-resistant epilepsy. Most SUDEP cases
+occur in bed at night and are preceded by a generalized tonic-clonic
+seizure (GTCS) and. Dravet syndrome (DS) is a severe childhood-onset
+epilepsy commonly caused by mutations in the \emph{SCN1A} gene. Affected
+individuals suffer from refractory seizures and an increased risk of
+SUDEP. Here, we demonstrate that mice with the
+\emph{Scn1a\textsuperscript{R1407X/+}} loss-of-function mutation
+experience more spontaneous seizures and SUDEP during the early night.
+In DS mice we evaluate effects of long-term ketogenic diet (KD)
+treatment on mortality and seizure frequency. DS mice showed high
+premature mortality (44\% survival by P60) that was associated with
+increased spontaneous GTCSs 1-2 days prior to death. KD treated mice had
+a significant reduction in mortality (86\% survival by P60) compared to
+mice fed a control diet. Interestingly, increased survival was not
+associated with decreased spontaneous non-fatal seizures. Further
+studies are needed to determine how KD confers protection from SUDEP.
+Moreover, our findings implicate time of day as a factor influencing the
+occurrence of seizures and SUDEP. DS mice, though nocturnal, are more
+likely to have SUDEP at night, suggesting that the increased incidence
+of SUDEP at night in DS mice may not be solely due to sleep.
+
+\textbf{INTRODUCTION}
+
+Sudden unexpected death in epilepsy (SUDEP) is estimated to occur in
+\textasciitilde{}17\% of patients with epilepsy\textsuperscript{1}. This
+number can drastically increase to 50\% in patients with poorly
+controlled and severe epilepsy\textsuperscript{2}. Although the
+mechanisms underlying SUDEP are not fully understood, an increasing body
+of evidence suggests SUDEP is due to seizure-induced cardiorespiratory
+dysfunction\textsuperscript{3,4}. However, little is known about the
+circumstances leading up to SUDEP. A strong association with sleep has
+been documented in a number of studies\textsuperscript{3,5}. Although a
+significant majority of patients are found in bed in the prone position
+at the time of death\textsuperscript{6-8}, the occurrence of SUDEP
+during sleep varies widely among published case
+studies\textsuperscript{9}. This suggests that other circadian factors
+may be relevant.
+
+Dravet Syndrome (DS) is a devastating epileptic encephalopathy of
+childhood-onset that typically manifests as febrile seizures in the
+first year of life and progresses to refractory
```

Comment:
refractory -> (intractable or untreatable)

refractory to me almost sounds like a good thing, but it is not.